### PR TITLE
Add docs to all Tensors

### LINKED ISSUE #1764: Only FloatTensor has docstrings
Steps to repro:

```
>>> help(torch.FloatTensor())                                                                                                 
>>> help(torch.DoubleTensor())
```

This doesn't seem good. Is there a good reason we didn't attach docs to everything? I think it would be a simple matter of modifying `torch/_tensor_docs.py` to loop over all types (you'd also want to NOT add the docstring if the method doesn't exist). CC @apaszke who originally added docstring support.

### PR DESCRIPTION
**Pull request by vlasenkov**
Resolves #1764

### CODE REVIEW

File: torch/_tensor_docs.py [hunk 1]
```diff
@@ -3,135 +3,159 @@
...
 
 See :func:`torch.acos`
 """)
 
-add_docstr(torch._C.FloatTensorBase.acos_,
-           """
-acos_() -> Tensor
+add_docstr_all('acos_',
+               """
+acos_() -> {0}
```
Comment (apaszke) [inline torch/_tensor_docs.py]: So if you look at the [online docs](http://pytorch.org/docs/), you'll see that we have them for a single thing called `torch.Tensor`, so we don't repeat them for each type. This is why these return types were like that, and I think it's better to keep them that way

**Review by apaszke — COMMENTED** [review]
Looks great, thanks! Just one comment

File: torch/_tensor_docs.py (hunk 1 above)
Comment (vlasenkov) [inline torch/_tensor_docs.py]: How do you generate these online docs? For example: I've added docs for all tensor types, will it lead to separate entries in the docs? Or you somehow join them?

Comment (apaszke) [inline torch/_tensor_docs.py]: We just take the docs from `FloatTensor` and put them under `torch.Tensor`

Comment (vlasenkov) [inline torch/_tensor_docs.py]: Ok. And for storages I'm to do the same?

Comment (apaszke) [inline torch/_tensor_docs.py]: Yup

**Comment by soumith** [comment]
thanks @vlasenkov !

**Pull request merged**